lessonbuilder/tool/src/webapp/templates/ShowPage.html

```diff
@@ -506,7 +506,7 @@ <h4><span rsf:id="msg=simplepage.layout.elements"></span></h4>
 			<h2 rsf:id="msg=simplepage.maincontent" class="offscreen" id="maintablelabel"></h2>
 <!-- WARNING: javascript for add-break-section adds section, ul, li, and sectionedit markup, so if you change here, keep it in sync -->
 			<div rsf:id="sectionWrapper:"><!-- start section wrapper -->
-			<h3 rsf:id="sectionHeader" id="sectionHeader" class="sectionHeader"><span aria-hidden="true" class="collapseIcon fa-caret-down"></span><span rsf:id="sectionHeaderText" class="sectionHeaderText"></span><span class="toggleCollapse" rsf:id="msg=simplepage.clickToCollapse"></span></h3>
+			<h3 rsf:id="sectionHeader" id="sectionHeader" class="sectionHeader" tabindex="0"><span aria-hidden="true" class="collapseIcon fa-caret-down"></span><span rsf:id="sectionHeaderText" class="sectionHeaderText"></span><span class="toggleCollapse" rsf:id="msg=simplepage.clickToCollapse"></span></h3>
```

Comment:
I infer you mean something like this (where the proposed changes to javascript can be revoked):

<h3 rsf:id="sectionHeader" id="sectionHeader" class="sectionHeader"><button type="button" style="background-color:inherit; padding: 0 3px; border: none"><span aria-hidden="true" class="collapseIcon fa-caret-down"></span><span class="toggleCollapse" rsf:id="msg=simplepage.clickToCollapse"></span></button><span rsf:id="sectionHeaderText" class="sectionHeaderText"></span></h3>

I assume for a screen reader that the 'toggleCollapse' span should be within the button and that because the button is interactive, the button should not be aria-hidden, but the icon should. 

Let me know what you advise, and I'll push those changes to overwrite my initial commit.

Reply:
Writing this stuff from scratch isn't fun. Here is a good example:

https://getbootstrap.com/docs/4.3/components/collapse/

I don't recommend using tabindex to make items tabbable. I think it's usually better to just use the right element, e.g., a button when you want someone to click

Don't change aria-label dynamically. Do use aria-expanded=true/false to represent whether it's expanded

Remove aria-hidden once you have an aria-label

Reply:
Noted and thanks. For what it's worth, the aria-hidden=true in my last h3 snippet was not intended (especially with an aria-label).